cert/storage/disk.go

```diff
@@ -81,7 +87,7 @@ func (fb *FileBackend) GetPaths() []string {
 // Load loads any PKI that this backend has on disk, checking that permissions are correct- erroring
 // if content is missing that is expected, or permissions don't align.
 func (fb *FileBackend) Load() (*x509.Certificate, *tls.Certificate, error) {
-	log.Debugf("loading PKI material for %s", fb)
+	fb.log.Debug().Msg("loading pki backend")
```

Comment:
nit: the previous entry (and below) had PKI uppercased, which seems right.

Reply:
I'm unsure about this. On my initial pass I just lowercased everything, but you're right that 'pki' looks weird.  I'm not really a stickler on this though so I'll go ahead and s/pki/PKI/

Reply:
..especially now that I noticed "CA" is also already an exception to that.